app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,13 @@
+class CommentsController < ApplicationController
+  def create
```

Comment:
- [ ] According to the project requirement,  the comments controller should have` new `and `create` actions similar to the `posts_controllers.rb`. The **new** action is missing here. Please fix it to make your app works as expected 👍